python-sdk/pachyderm_sdk/api/__init__.py

```diff
@@ -0,0 +1 @@
+from .pfs import _additions as __pfs_additions
```

Comment:
This should be moved to the top level `__init__.py` Future work would be to revisit these additions